### ISSUE TITLE
Feature request: Image alt text

### ISSUE THREAD

**Issue by filcuk** [silverbulletmd/silverbullet#820]
This is typically implemented as: `![Alt text](URL or file path)`
A title or caption would also be very useful: `![alt text](image url "image Title")`

It would work alongside https://github.com/silverbulletmd/silverbullet/issues/493: `![alt text](image url "image Title" | resolution)`
(note that above issue suggests a slightly different arrangement for alt text & resolution in comments, either is fine).

**Comment by GiantCrocodile** [comment]
Image alt text is already working but caption/title is missing it looks like, I reported something similar:

https://github.com/silverbulletmd/silverbullet/issues/862
